resources/Materials/TestSuite/stdlib/definition/within2_color3.mtlx

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+  <nodedef name="adsk:ND_within2_color3" node="within2" version="1.0" isdefaultversion="true" nodegroup="BenoitsTest" namespace="adsk">
```

Comment:
Sorry I missed this, can we remove Benosts name and the adsk namespace for both files :).